build/root/BUILD.root

```diff
@@ -65,6 +65,16 @@ filegroup(
         "//examples:all-srcs",
         "//federation:all-srcs",
         "//hack:all-srcs",
+        "//kubernetes/cluster:all-srcs",
+        "//kubernetes/docs:all-srcs",
+        "//kubernetes/examples:all-srcs",
+        "//kubernetes/federation/cluster:all-srcs",
+        "//kubernetes/federation/develop:all-srcs",
+        "//kubernetes/hack:all-srcs",
+        "//kubernetes/test/e2e/testing-manifests:all-srcs",
+        "//kubernetes/test/images:all-srcs",
+        "//kubernetes/test/kubemark:all-srcs",
+        "//kubernetes/third_party/htpasswd:all-srcs",
```

Comment:
hummm, those were wrote by `./hack/update-bazel.sh`

Reply:
you probably have a kubernetes/ dir in the root of your repo, probably due to extracting a `kubernetes.tar.gz` there. remove it and re-run `update-bazel.sh`.